plugin/storage/integration/token_propagation_test.go

```diff
@@ -89,6 +102,19 @@ func TestBearTokenPropagation(t *testing.T) {
 	defer srv.Shutdown(context.Background())
 
 	go createElasticSearchMock(srv, t)
+	// Wait for http server to start
+	time.Sleep(100 * time.Millisecond)
+
+	// Path relative to plugin/storage/integration/token_propagation_test.go
+	cmd := exec.Command("../../../cmd/query/query-linux", "--es.server-urls=http://127.0.0.1:9200", "--es.tls=false", "--query.bearer-token-propagation=true")
```

Comment:
Could you please revert this. It's better to run the query from outside the test.

Reply:
The query service must be started after the elasticsearch server because the version check is done when the client is initialised (ie on startup).  If you start the query service before running the test, as it was before, the query service exits with an error.